ipn/localapi/localapi.go

```diff
@@ -1904,6 +1905,31 @@ func (h *Handler) serveTKAStatus(w http.ResponseWriter, r *http.Request) {
 	w.Write(j)
 }
 
+func (h *Handler) serveSetGUIVisible(w http.ResponseWriter, r *http.Request) {
+	if !h.PermitWrite {
+		http.Error(w, "GUI visible access denied", http.StatusForbidden)
+		return
+	}
+	if r.Method != httpm.POST {
+		http.Error(w, "use POST", http.StatusMethodNotAllowed)
+		return
+	}
+
+	type guiVisible struct {
```

Comment:
mabye `setGUIVisibleRequest` ? We might want to export these later but for now a local type is fine.

But let's document the two fields for future readers at least. We remember now but we'll forget in 6 months :)

Reply:
Renamed and documented